### DISCUSSION TITLE
Simple test menu using most of Nano's storage space

### DISCUSSION THREAD

**Discussion by Intechgreater** [davetcc/tcMenu#477] [General]
A simple test menu with only a single analog menu item generated in TcMenu Designer 4.2.2 and compiled in Arduino IDE for an Arduino Nano running a 128x64 OLED display via the Greiman SSD1306ASCII library uses 30092 bytes (97%) of program storage space! 
Is that unusual?
Is there anything that can be changed to greatly reduce that?

**Comment by davetcc** [comment]
We no longer really recommend using tcMenu on ATMEGA328, with only 32K Flash it doesn't have enough space. I've stopped even testing on Uno some time ago. 

Most under-clocked ARM Cortex0 boards are able to operate with exceptionally low power anyway. 

Closing out this now.
